Digital Innovation KPIs

sprice opened the pull request:
In #59 we talked about both power capacity and production. This PR currently includes both.

I ensured no lint errors in these files. I see the codebase has some elsewhere. `npm run build` works.

I'm happy to update the PR to remove the change to `.gitignore` which ignores `.claude` directories if that's a preference.

Fixes #59 

<img width="947" alt="image" src="https://github.com/user-attachments/assets/ca520167-e6e6-418d-a7f6-9eb6a5959d95" />

xrendan commented:
I'd love for the Claude directory to be in the repo so everyone benefits. 

Where are you grabbing the data from?

sprice commented:
Each dashboard has a link to the StatsCan page.

I downloaded a CSV for monthly production and converted that to JS on. And I screencapped annual capacity and converted to json. 

I'd prefer to open a new PR along the lines of "Add some basic LLM support". CLAUDE.md file and .claude directory, cursor rules files, other things that other devs would like (cline, windsurf etc)
